aperturedb/cli/configure.py

```diff
@@ -149,28 +160,49 @@ def create(
             style="bold yellow")
         raise typer.Exit(code=2)
 
-    if interactive:
-        db_host = typer.prompt(f"Enter {APP_NAME} host name", default=db_host)
-        db_port = typer.prompt(
-            f"Enter {APP_NAME} port number", default=db_port)
-        db_username = typer.prompt(
-            f"Enter {APP_NAME} username", default=db_username)
-        db_password = typer.prompt(
-            f"Enter {APP_NAME} password", hide_input=True, default=db_password)
-        db_use_rest = typer.confirm(
-            f"Use REST [Note: Only if ApertureDB is setup to recieve http requests]", default=db_use_rest)
-        db_use_ssl = typer.confirm(
-            f"Use SSL [Note: ApertureDB's defaults do not allow non SSL traffic]", default=db_use_ssl)
-
-    gen_config = Configuration(
-        name=name,
-        host=db_host,
-        port=db_port,
-        username=db_username,
-        password=db_password,
-        use_ssl=db_use_ssl,
-        use_rest=db_use_rest
-    )
+    if from_json:
+        json_str = os.getenv("APERTUREDB_JSON")
```

Comment:
So as of now, after I copy JSON from cloud , I will need to past it in a cell, fill my password and use that string ?

Reply:
You could, but my recommendation would be to prepare the JSON string in a more private editor.

Reply:
OK - i was just thinking if it makes sense to prompt passwd and create this fully complete JSON in quick start or we explain that users have to paste a json with their passwd

Reply:
See https://github.com/aperture-data/Cookbook/pull/13 for the simplified Quickstart.